core/src/main/java/com/capitalone/dashboard/model/Configuration.java

```diff
@@ -0,0 +1,61 @@
+package com.capitalone.dashboard.model;
+
+import java.util.Base64;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "configuration")
+public class Configuration extends BaseModel {
+
+	private final static String PREFIX = "BASE64:";
+
+	public Configuration(String collectorName, Set<Map<String, String>> info) {
+		super();
+		this.collectorName = collectorName;
+		this.info = info;
+	}
+
+	public Set<Map<String, String>> decryptOrEncrptInfo() {
```

Comment:
@Satheesh-Balachandran  can we use existing Encryption utils for encryption and decryption. That way it is easy to secure confidential information

Reply:
@nireeshT Encryption utils required api key to encrypt data. Since, this method will be used by the collectors to decrypt the password to fetch data from tools, api key will not be available here. So its better to inject encryption here and also not to repeat code in each collectors for decryption.